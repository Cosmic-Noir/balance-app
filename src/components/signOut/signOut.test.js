import React from "react";
import ReactDom from "react-dom";
import renderer from "react-test-renderer";
import SignOut from "./signOut";
import { BrowserRouter } from "react-router-dom";

describe(`SignOut Component`, () => {
  it(`Renders without crashing`, () => {
    const div = document.createElement("div");
    ReactDom.render(
      <BrowserRouter>
        <SignOut />
      </BrowserRouter>,
      div
    );
    ReactDom.unmountComponentAtNode(div);
  });

  it(`Renders the UI as expected`, () => {
    const tree = renderer.create(
      <BrowserRouter>
        <SignOut />
      </BrowserRouter>
    ).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
