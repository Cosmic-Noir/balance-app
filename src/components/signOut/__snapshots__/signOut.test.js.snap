// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignOut Component Renders the UI as expected 1`] = `[Function]`;
